demo/meeting/back-end-resources/lambda-layer/nodejs/node_modules/utils.js

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: Use `const` and `let` for consistency. Again, linters and formatters will help to fix minor syntax styles.